CONTRIBUTION.md

```diff
@@ -0,0 +1,267 @@
+
+* **FRCore**: The FRCore module acts as the foundational layer for all other modules within the SDK. It 
+provides a set of common functions and utilities that are shared across the SDK, ensuring 
+consistency and reducing code duplication.
+
+* **FRDeviceBinding**: This module allows developers to cryptographically bind a mobile device to a user account. Registered devices generate a key pair and a key ID. The SDK sends the public key and key ID to AM for storage in the user’s profile. With this capability, developers using FRAuth module with authentication journeys can verify ownership of the bound device by requesting that it signs a challenge using the private key.
+
+* **FRFacebookSignIn**: This module is part of the Social Login feature. It allows developers to enable sign-in through Facebook. FRFacebookSignIn depends on FBSDKLoginKit, and uses Facebook's SDK to perform authorization following Facebook's protocol
+
+* **FRGoogleSignIn**: This module is part of the Social Login feature that allows a user to sign-in through Google. FRGoogleSignIn depends on GoogleSignIn, and uses Google's SDK to perform authorization following Google's protocol.
+
+* **FRProximity**: FRProximity is the module that allows developers to additionally collect device information with FRDeviceCollector in FRAuth. It provides a subset of tools and functionalities related to proximity features (such as location, and BLE) in the device. Some functionalities requires user's consent.
+
+* **FRTestHost**: This module is dedicated to the SDK's tests. It provides the host app and shared test resources to enable execution of tests within the SDK.
+
+* **FRUI**: The FRUI module provides a collection of pre-built UI components designed to expedite 
+the development of prototype applications. These UI elements integrate directly with the 
+forgerock-auth module and are specifically designed to facilitate rapid prototyping with the 
+Journey framework. This module is deprecated.
+
+### Running Tests
+
+Unit testing is essential for software development. It ensures individual code components work 
+correctly, catches bugs early, and improves code reliability. This section explains how to run  
+tests for all SDK modules.
+
+#### Execute tests
+
+   ```bash
+          xcodebuild test \
+          -scheme FRTestHost \
+          -workspace e2e/FRExample.xcworkspace \
+          -configuration Debug \
+          -destination 'platform=iOS Simulator,name=iPhone 16,OS=18.1' \
+          -derivedDataPath DerivedData \
+          TEST_TARGET_SIGNING=YES \
+          -enableCodeCoverage YES \
+          -resultBundlePath TestResults
+   ```
+
+The `destination` should reflect the target device/emulator and iOS version available.
+
+
+### Build API Reference Documentation
```

Comment:
I think we should take this section out from the contribution guide. 
Currently we generate the API reference docs during the release cycle. 
We eventually will automated this, but IMO contributors should just be asked to add the docs in the code...

Reply:
We discussed this and agreed that when we automate the generation of the API reference docs we will update the guideline as needed.